go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/vishnuchalla/go-commons
```

Comment:
should this be `cloud-bulldozer/go-commons` ?

Reply:
Acknowledged. Changed as suggested.